app/components/auk/datepicker.hbs

```diff
@@ -1,9 +1,7 @@
 <div class="auk-input-with-icon {{@class}}" {{did-update this.updateEnable this.enable}}>
   {{!-- template-lint-disable no-invalid-interactive --}}
   {{!-- TODO: figure out a more accessible solution than a clickable div --}}
-  <div {{on "click" this.openDatepicker}}>
-    <Auk::Icon @name="calendar"/>
-  </div>
+  <Auk::Icon @name="calendar"/>
```

Comment:
Kan je hier wat extra duiding bij geven? Hier lijkt toch wel echt functionaliteit te zijn veranderd. De comment & linting rule exception die blijven staan is, zou ook weg mogen indien bedoeld?

Reply:
Terechte vraag! Die `on "click"` was eigenlijk een fix voor het feit dan icons binnen input elementen op zich aanklikbaar waren en in het geval van bv. datepicker de click functionaliteit van de onderliggende flatpickr tegen hielden. Die `on "click"` hadden ze dus hier aan toegevoegd, maar dit is op zich makkelijker op te lossen door alle icons binnen input elementen `pointer-events: none;` te geven via CSS (wat op zich ook logischer is).

Reply:
Die disabled linting rule + commentaar heb ik ondertussen ook verwijderd.

Reply:
:ok_hand: